sys-auth/pam_u2f/pam_u2f-1.1.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools flag-o-matic pam
+
+DESCRIPTION="PAM module for FIDO2 and U2F keys"
+HOMEPAGE="https://github.com/Yubico/pam-u2f"
+SRC_URI="https://developers.yubico.com/${PN/_/-}/Releases/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug"
+
+RDEPEND="
+	dev-libs/libfido2
```

Comment:
I also realized that `libfido2` has subslots now. Let's be proactive and add a [slot operator](https://devmanual.gentoo.org/general-concepts/dependencies/#slot-operators) as follows:

```
dev-libs/libfido2:0=
```